src/gui/sharelinkwidget.cpp

```diff
@@ -473,6 +485,20 @@ void ShareLinkWidget::slotPushButtonCopyLinkPressed()
 #endif
 }
 
+void ShareLinkWidget::slotPushButtonMailLinkPressed()
+{
+    QString fileName = _sharePath.mid(_sharePath.lastIndexOf('/') + 1);
+
+    QDesktopServices::openUrl(QUrl(QString(
+            "mailto: "
+            "?subject=%1 has shared %2 with you"
+            "&body=%3").arg(
+            _account->credentials()->user(),
```

Comment:
Is _account->credentials()->user() the sender's username as entered in the application or is it their display name provided by the server?